apps/worker/tasks/base.py

```diff
@@ -49,6 +50,9 @@
 
 log = logging.getLogger("worker")
 
+# Sentinel object to distinguish "not provided" from "explicitly None" for max_retries
+_NOT_PROVIDED = object()
```

Comment:
this is a little bit weird to make a comparison to `object()`. I'd prefer is this were a singleton class or some other primitive